.github/workflows/py-workflow.yml

```diff
@@ -159,7 +159,6 @@ jobs:
         echo "Please check report here: https://sonarcloud.io/project/overview?id=${{ github.repository_owner }}_${{ github.event.repository.name }}"
 
     - name: Raise version
-      if: ${{ inputs.deploy == 'true' }}
       run: bump2version build setup.py --no-tag --no-commit
```

Comment:
the 2 steps can be merged instead of using the `--no-commit`:

```
- name: Raise version and commit
  run: |
    git config --global --add safe.directory $(pwd)
    git config --global user.name ${{ secrets.auto_commit_user }}
    git config --global user.email ${{ secrets.auto_commit_mail }}
   bump2version build setup.py --no-tag
```

Reply:
The step `Sync setup.py` also changes files, but it seems that is changing the value to the one that is already there IF bump2version didn't update `setup.py`. Do you know in which conditions it doesn't?